app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -32,6 +33,8 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {weather.current.windGust > 15 &&
```

Comment:
What is this comparison doing? Will the wind gust information only be displayed if it is above 15?

Reply:
yes, the information is displayed only for the speeed>15